### DISCUSSION TITLE
[Python | Timezone] How to enable timezone in the Python API?

### DISCUSSION THREAD

**Discussion by codedecde** [microsoft/Recognizers-Text#2064] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
How to use the DateTimeModel for parsing timezones.
Eg: For Sentence "Let's meet at 15:00 GMT Saturday", the following model

```[python]
text = "Let's meet at 15:00 GMT Saturday"
model = DateTimeRecognizer(Culture.English).get_datetime_model()
predictions = model.parse(text)

print(len(predictions))  # 2
print(predictions[0].text)  # 15:00
print(predictions[1].text)  # Saturday
```
Yields 15:00 and Saturday. 
How would it be possible to extract (15:00 GMT Saturday) as a single entity ?
It seems like there is a way to do it by setting DateTimeOptions.ENABLE_PREVIEW, but can't figure out how exactly to do this.
An example would be amazing. 
Thank you

**Comment by tellarin** [comment, marked as answer]
Unfortunatelty Timezone preview is only available in .NET. It will propagate to Python once its enabled by default in .NET.
